src/organizations/OrganizationEntity.js

```diff
@@ -40,6 +41,24 @@ export class OrganizationEntity extends Entity {
 		return this._entity && this._entity.properties && this._entity.properties.endDate;
 	}
 
+	get isCompletionTracked() {
+		if (this._entity && this._entity.hasActionByName('track-completion')) {
```

Comment:
there is actions object that defines names of actions. It is better to add actions for tracking completion and display progress there. you could reference them in new methods for example:`_entity.hasActionByName(actions.trackCompletion)`.